<html>
<head>
    <!--plotly scripts -->
<script src="{{ url_for('static', filename='js/third-party/plotly/dist/plotly.min.js') }}"></script>
    <!--bootstrap scripts -->
	    <link href= "{{ url_for('static',
	filename='js/third-party/bootstrap/dist/css/bootstrap.min.css')}}" rel="stylesheet">
    <script src="{{ url_for('static', filename='js/third-party/jquery/dist/jquery.min.js') }}"></script>
	<script type="text/javascript" src="{{ url_for('static', filename='js/third-party/bootstrap/dist/js/bootstrap.min.js') }}">
	</script>


</head>
    <!-- logic (Test Flash) -->
	{% with messages = get_flashed_messages() %}
		{% if messages %}
			{% for message in messages %}
				<div class="alert alert-warning alert-dismissible" role="alert">
					<button type="button" class="close" data-dismiss="alert" aria-label="Close">x</button>
					{{message}}
				</div>

			{% endfor %}
		{% endif %}
	{% endwith %}

<body>

<div id="aboutTextCompare">

            <!-- Button trigger modal -->
        <button type="button" class="btn btn-success btn-sm" data-toggle="modal" data-target="#textCompare">
          How are these scores calculated?
        </button>

        <!-- Modal -->
        <div class="modal fade" id="textCompare" tabindex="-1" role="dialog" aria-labelledby="textCompareLabel">
          <div class="modal-dialog" role="document">
            <div class="modal-content">
              <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
                <h4 class="modal-title" id="textCompareLabel">About TextCompare</h4>
              </div>
              <div class="modal-body">
                In order to compare documents with their percent similarity, we transform
                documents into vectors with a naive implementation of the doc2vec  algorithm. and then compute
                  After the documents are converted into vectors, we take the cosine similarity between each text and
                  the vector of the selected TextCompare document (e.g. Origin of The Species).
              </div>
            </div>
          </div>
        </div>

</div>

<form class="form inline" action="https://geco.iplantcollaborative.org/scke/coge_scifi/" method="post">
<div class="form-group">
  <label for="buttongroup">Select a text to compare:</label>
  <div id="buttongroup">
     <div class="col-xs-6 col-sm-4">
    <select class="form-control" id="corpus" name="corpus">
        <option name="darwin" value="darwin">On the Origin of Species</option>
        <option name="yeast" value="yeast">Yeast</option>
        <option name="mouse" value="mouse">The Dancing Mouse</option>
        <option name="brain_speech" value="brain_speech">The Brain & The Voice</option>
        <option name="grecoroman" value="grecoroman">Outlines of Greek & Roman Medicine</option>
      <option name="startrek" value="startrek">Star Trek corpus</option>
        <option name="mars" value="mars">Guilliver of Mars</option>
        <option name="last_evolution" value="last_evolution">The Last Evolution</option>
         <option name="youth" value="youth">Youth</option>
        <option name="frankenstein" value="frankenstein">Frankenstein</option>
        <option name="sherlock" value="sherlock">Sherlock Holmes</option>
    <option name="austen" value="austen">Pride and Prejudice</option>
    <option name="bible" value="bible">The Bible</option>
         {% for user_input in eligible_papers %}
             <option name="{{user_input[0]}}" value="{{user_input[0]}}">{{user_input[3]}}</option>
		  	{% endfor %}

       </select>
       </div>


  <button type="submit" id="litbutton" class="btn btn-default" data-toggle="modal" data-target="#myModal">Update</button>


   </div>
  </div>
</form>

<div id="myDiv" style="width: 690px; height: 650px;"><!-- Plotly chart will be drawn inside this DIV --></div>
<div id="clickInfo"> </div>


<script>
    var myPlot = document.getElementById('myDiv'),
        clickInfo = document.getElementById('clickInfo'),

        data = [{
    x: ['2013, Mouhu ', '2012, Lou ', '2014, Thompson ', '2008, Freeling ', '2013, Moon ', '2009, Bolduc ', '2017, Bernal ', '2008b, Lyons ', '2010, Ron ', '2015, Ming ', '2009, Jurkuta ', '2013, Hofberger ', '2008, Lyons ', '2016, Reinprecht ', '2008, Ming ', '2011, Severin ', '2013, Ludwig ', '2011, Woodhouse ', '2016, Panchy ', '2016, Best ', '2016, Bewick ', '2014, Parker ', '2014, Burgess ', '2013, Wang ', '2014, Zulawski ', '2014, Krishnakumar ', '2016, Ndikumana ', '2016, Booker ', '2016, Zhang ', '2010, Takata ', '2016, Trujillo ', '2014, Jiao ', '2010, Mesplet ', '2014, Guo ', '2015, Bruenn ', '2016, Qiao ', '2013, Colasuonno ', '2012, Revanna ', '2011, Donoghue ', '2013, Książkiewicz ', '2016, Gutsche ', '2014, Ganapathy ', '2016, Pandey ', '2013, Rabier ', '2014, Martin ', '2012, Lee ', '2016, Xu ', '2010, Kane ', '2010, Shi ', '2012, Reneker ', '2016, Li ', '2009, Proost ', '2012, Wang ', '2015, Wang ', '2011, Zuccolo ', '2016, Rao ', '2012, Field ', '2012, Tang ', '2013, Cheng ', '2015, Pavlopoulos ', '2012, Albuquerque ', '2016, Wang ', '2013, Guo ', '2017, Liu ', '2014, Jaratlerdsiri ', '2016, Chen ', '2012, Zhang ', '2017, Fan ', '2014, Boyd ', '2011, Gustin ', '2009, Barker ', '2013, Campoli ', '2011, Schnable ', '2010, Woodhouse ', '2013, Subramaniam ', '2015, Mohammadin ', '2016, Mauro...', '2016, Wyrwa ', '2014, Wang ', '2016, Fister ', '2015, Hofberger ', '2011, Tang ', '2014, Drillon ', '2014, de ', '2012b, Zhang ', '2014, Cannarozzi ', '2016, Liu ', '2014, Trujillo ', '2010, Sanzol ', '2014, Chen ', '2011, Abeel ', '2015, Tang ', '2011b, Schnable ', '2016, Zhao ', '2013, Waters ', '2011, Lyons ', '2016, Mykles ', '2011, Mondego ', '2012, Cheng ', '2012b, Tang ', '2015, Morioka ', '2011, Soderlund ', '2013, Turco ', '2014, Hofberger ', '2014, Zheng ', '2012, Sankoff ', '2015, Berke ', '2013, Yang ', '2010, Hand ', '2015b, Hofberger ', '2015, Zheng ', '2013, Froyd ', '2013, Mauro...', '2011, Hudson ', '2011, Van ', '2014, Książkiewicz ', '2012, Wiszniewski ', '2011, Goff ', '2016, Książkiewicz ', '2015, Devis ', '2013, Jung ', '2015, Przysiecka ', '2011, Reineke ', '2014, Reichardt ', '2016, Zheng ', '2012, Chen ', '2015, Haug-...', '2016, Scaglione ', '2014, Coman ', '2016, Boyd ', '2012, Nussbaumer ', '2013b, Zheng ', '2015, Kang ', '2017, Rendó...', '2015, Zhang ', '2012, Zheng ', '2014, Hyung ', '2013, Zheng ', '2013, Qin ', '2014, Masonbrink ', '2015, Perei...', '2014, Guimaraes ', '2014, Delaux ', '2012, Ranjan ', '2016, Baek ', '2013, Mühlhausen ', '2016, Wu ', '2016, Nelson ', '2016, Rupert ', '2013, Chen ', '2014, Louis ', '2010, Muffato ', '2014, Proost ', '2016, Lee ', '2014, Kagale ', '2016, Yerramsetty ', '2016, Studer '],
        y: [3.2, 3.95, 4.590000000000001, 4.83, 4.88, 5.29, 5.390000000000001, 5.47, 5.47, 5.47, 5.65, 6.069999999999999, 6.43, 6.54, 6.569999999999999, 6.63, 6.660000000000001, 6.81, 6.890000000000001, 7.090000000000001, 7.090000000000001, 7.1, 7.37, 7.5600000000000005, 7.95, 8.469999999999999, 8.780000000000001, 8.870000000000001, 8.9, 9.19, 9.27, 9.65, 10.0, 10.03, 10.03, 10.16, 10.32, 10.45, 10.54, 10.56, 10.56, 10.66, 10.66, 10.76, 10.84, 10.86, 10.94, 11.020000000000001, 11.03, 11.19, 11.19, 11.21, 11.27, 11.37, 11.39, 11.42, 11.49, 11.53, 11.540000000000001, 11.63, 11.86, 11.86, 11.899999999999999, 11.95, 12.11, 12.11, 12.120000000000001, 12.21, 12.23, 12.24, 12.27, 12.42, 12.44, 12.45, 12.5, 12.590000000000002, 12.659999999999998, 12.75, 12.879999999999999, 12.94, 12.959999999999999, 12.97, 13.020000000000001, 13.03, 13.04, 13.15, 13.170000000000002, 13.19, 13.569999999999999, 13.58, 13.600000000000001, 13.62, 13.68, 13.719999999999999, 13.850000000000001, 13.88, 14.21, 14.299999999999999, 14.469999999999999, 14.540000000000001, 14.549999999999999, 14.71, 14.899999999999999, 15.02, 15.160000000000002, 15.28, 15.290000000000001, 15.340000000000002, 15.49, 15.6, 15.690000000000001, 15.72, 15.93, 15.959999999999999, 15.98, 16.05, 16.16, 16.43, 16.46, 16.73, 16.75, 16.78, 16.939999999999998, 17.0, 17.169999999999998, 17.24, 17.29, 17.29, 17.51, 17.75, 17.8, 17.93, 18.23, 18.42, 18.57, 18.62, 19.08, 19.12, 19.53, 19.62, 19.7, 20.0, 20.49, 20.61, 20.75, 20.87, 21.0, 21.099999999999998, 21.54, 21.91, 22.400000000000002, 23.01, 23.53, 23.880000000000003, 24.05, 24.13, 24.73] ,
        name: ['<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3809533">Mouhu K, Kurokura T, Koskela EA, Albert VA, Elomaa P, Hytönen T (2013 Sep 13). The Fragaria vesca Homolog of SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1 Represses Flowering and Promotes Vegetative Growth. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3809533</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3406906">Lou P, Wu J, Cheng F, Cressman LG, Wang X, McClung CR (2012 Jun 8). Preferential Retention of Circadian Clock Genes during Diploidization following Whole Genome Triplication in Brassica rapa. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3406906</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4311206">Thompson BE, Basham C, Hammond R, Ding Q, Kakrana A, Lee TF, Simon SA, Meeley R, Meyers BC, Hake S (2014 Dec 2). The dicer-like1 Homolog fuzzy tassel Is Required for the Regulation of Meristem Determinacy in the Inflorescence and Vegetative Growth in Maize. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4311206</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593585">Freeling M, Lyons E, Pedersen B, Alam M, Ming R, Lisch D (2008 Dec). Many or most genes in Arabidopsis transposed after the origin of the order Brassicales. Genome Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593585</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4239027">Moon J, Skibbe D, Timofejeva L, Rachel Wang CJ, Kelliher T, Kremling K, Walbot V, Zacheus Cande W (2013 Oct 5). Regulation of cell divisions and differentiation by MALE STERILITY32 is required for anther development in maize. The Plant journal : for cell and molecular biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4239027</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714931">Bolduc N, Hake S (2009 Jun). The Maize Transcription Factor KNOTTED1 Directly Regulates the Gibberellin Catabolism Gene ga2ox1. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714931</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5363822">Bernal P, Allsopp LP, Filloux A, Llamas MA (2017 Jan 3). The Pseudomonas putida T6SS is a plant warden against phytopathogens. The ISME journal. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5363822</a>', '<a href="http://www.ncbi.nlm.nih.gov/pubmed/18952863">Lyons E, Pedersen B, Kane J, Alam M, Ming R, Tang H, Wang X, Bowers J, Paterson A, Lisch D, Freeling M (2008 Dec). Finding and comparing syntenic regions among Arabidopsis and the outgroups papaya, poplar, and grape: CoGe with rosids.. Plant physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/18952863</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2867206">Ron M, Alandete Saez M, Eshed Williams L, Fletcher JC, McCormick S (2010 May 15). Proper regulation of a sperm-specific cis-nat-siRNA is essential for double fertilization in Arabidopsis. Genes & Development. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2867206</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4867222">Ming R, VanBuren R, Wai CM, Tang H, Schatz MC, Bowers JE, Lyons E, Wang ML, Chen J, Biggers E, Zhang J, Huang L, Zhang L, Miao W, Zhang J, Ye Z, Miao C, Lin Z, Wang H, Zhou H, Yim WC, Priest HD, Zheng C, Woodhouse M, Edger PP, Guyot R, Guo HB, Guo H, Zheng G, Singh R, Sharma A, Min X, Zheng Y, Lee H, Gurtowski J, Sedlazeck FJ, Harkess A, McKain MR, Liao Z, Fang J, Liu J, Zhang X, Zhang Q, Hu W, Qin Y, Wang K, Chen LY, Shirley N, Lin YR, Liu LY, Hernandez AG, Wright CL, Bulone V, Tuskan GA, Heath K, Zee F, Moore PH, Sunkar R, Leebens-Mack JH, Mockler T, Bennetzen JL, Freeling M, Sankoff D, Paterson AH, Zhu X, Yang X, Smith JA, Cushman JC, Paull RE, Yu Q (2015 Nov 2). The pineapple genome and the evolution of CAM photosynthesis. Nature genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4867222</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2729608">Jurkuta RJ, Kaplinsky NJ, Spindel JE, Barton MK (2009 Jul). Partitioning the Apical Domain of the Arabidopsis Embryo Requires the BOBBER1 NudC Domain Protein. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2729608</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3845643">Hofberger JA, Lyons E, Edger PP, Chris Pires J, Eric Schranz M (2013 Oct 29). Whole Genome and Tandem Duplicate Retention Facilitated Glucosinolate Pathway Diversification in the Mustard Family. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3845643</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593677">Lyons E, Pedersen B, Kane J, Alam M, Ming R, Tang H, Wang X, Bowers J, Paterson A, Lisch D, Freeling M (2008 Dec). Finding and Comparing Syntenic Regions among Arabidopsis and the Outgroups Papaya, Poplar, and Grape: CoGe with Rosids. Plant Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593677</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810023">Reinprecht Y, Pauls KP (2016 Mar 29). Microsomal Omega-3 Fatty Acid Desaturase Genes in Low Linolenic Acid Soybean Line RG10 and Validation of Major Linolenic Acid QTL. Frontiers in Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810023</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836516">Ming R, Hou S, Feng Y, Yu Q, Dionne-Laporte A, Saw JH, Senin P, Wang W, Ly BV, Lewis KL, Salzberg SL, Feng L, Jones MR, Skelton RL, Murray JE, Chen C, Qian W, Shen J, Du P, Eustice M, Tong E, Tang H, Lyons E, Paull RE, Michael TP, Wall K, Rice DW, Albert H, Wang ML, Zhu YJ, Schatz M, Nagarajan N, Acob RA, Guan P, Blas A, Wai CM, Ackerman CM, Ren Y, Liu C, Wang J, Wang J, Na JK, Shakirov EV, Haas B, Thimmapuram J, Nelson D, Wang X, Bowers JE, Gschwend AR, Delcher AL, Singh R, Suzuki JY, Tripathi S, Neupane K, Wei H, Irikura B, Paidi M, Jiang N, Zhang W, Presting G, Windsor A, Navajas-Pérez R, Torres MJ, Feltus FA, Porter B, Li Y, Burroughs AM, Luo MC, Liu L, Christopher DA, Mount SM, Moore PH, Sugimura T, Jiang J, Schuler MA, Friedman V, Mitchell-Olds T, Shippen DE, dePamphilis CW, Palmer JD, Freeling M, Paterson AH, Gonsalves D, Wang L, Alam M (2008 Apr 24). The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus). Nature. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836516</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3203428">Severin AJ, Cannon SB, Graham MM, Grant D, Shoemaker RC (2011 Sep 13). Changes in Twelve Homoeologous Genomic Regions in Soybean following Three Rounds of Polyploidy. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3203428</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815225">Ludwig Y, Zhang Y, Hochholdinger F (2013 Nov 1). The Maize (Zea mays L.) AUXIN/INDOLE-3-ACETIC ACID Gene Family: Phylogeny, Synteny, and Unique Root-Type and Tissue-Specific Expression Patterns during Development. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3815225</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3269863">Woodhouse MR, Tang H, Freeling M (2011 Dec 16). Different Gene Families in Arabidopsis thaliana Transposed in Different Epochs and at Different Frequencies throughout the Rosids. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3269863</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4972278">Panchy N, Lehti-Shiu M, Shiu SH (2016 Jun 10). Evolution of Gene Duplication in Plants. Plant Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4972278</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4972271">Best NB, Hartwig T, Budka J, Fujioka S, Johal G, Schulz B, Dilkes BP (2016 Jun 10). nana plant2 Encodes a Maize Ortholog of the Arabidopsis Brassinosteroid Biosynthesis Gene DWARF1, Identifying Developmental Interactions between Brassinosteroids and Gibberellins. Plant Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4972271</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987809">Bewick AJ, Ji L, Niederhuth CE, Willing EM, Hofmeister BT, Shi X, Wang L, Lu Z, Rohr NA, Hartwig B, Kiefer C, Deal RB, Schmutz J, Grimwood J, Stroud H, Jacobsen SE, Schneeberger K, Zhang X, Schmitz RJ (2016 Jul 25). On the origin and evolutionary consequences of gene body DNA methylation. Proceedings of the National Academy of Sciences of the United States of America. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987809</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256881">Parker N, Wang Y, Meinke D (2014 Oct 21). Natural Variation in Sensitivity to a Loss of Chloroplast Translation in Arabidopsis. Plant Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256881</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001403">Burgess D, Freeling M (2014 Mar 28). The Most Deeply Conserved Noncoding Sequences in Plants Serve Similar Functions to Those in Vertebrates Despite Large Differences in Evolutionary Rates. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4001403</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590777">Wang Y (2013 Jan 28). Locally Duplicated Ohnologs Evolve Faster Than Nonlocally Duplicated Ohnologs in Arabidopsis and Rice. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590777</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4112214">Zulawski M, Schulze G, Braginets R, Hartmann S, Schulze WX (2014 Jul 1). The Arabidopsis Kinome: phylogeny and evolutionary insights into functional diversification. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4112214</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383980">Krishnakumar V, Hanlon MR, Contrino S, Ferlanti ES, Karamycheva S, Kim M, Rosen BD, Cheng CY, Moreira W, Mock SA, Stubbs J, Sullivan JM, Krampis K, Miller JR, Micklem G, Vaughn M, Town CD (2014 Nov 20). Araport: the Arabidopsis Information Portal. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383980</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5124540">Ndikumana S, Pelin A, Williot A, Sanders JL, Kent M, Corradi N (2016 Jun 27). Genome analysis of Pseudoloma neurophilia: a microsporidian parasite of Zebrafish (Danio rerio). The Journal of eukaryotic microbiology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5124540</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291013">Booker MA, DeLong A (2016 Dec 29). Atypical Protein Phosphatase 2A Gene Families Do Not Expand via Paleopolyploidization. Plant Physiology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291013</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4832253">Zhang HX, Jin JH, He YM, Lu BY, Li DW, Chai WG, Khan A, Gong ZH (2016 Apr 15). Genome-Wide Identification and Analysis of the SBP-Box Family Genes under Phytophthora capsici Stress in Pepper (Capsicum annuum L.). Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4832253</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2887406">Takata N, Saito S, Saito CT, Uemura M (2010 May 1). Phylogenetic footprint of the plant clock system in angiosperms: evolutionary processes of Pseudo-Response Regulators. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2887406</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5125548">Trujillo JT, Beilstein MA, Mosher RA (2016 Jul 19). The Argonaute-binding platform of NRPE1 evolves through modulation of intrinsically disordered repeats. The New phytologist. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5125548</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4071528">Jiao Y, Paterson AH (2014 Aug 5). Polyploidy-associated genome modifications during land plant evolution. Philosophical Transactions of the Royal Society B: Biological Sciences. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4071528</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003645">Mesplet M, Echaide I, Dominguez M, Mosqueda JJ, Suarez CE, Schnittger L, Florin-Christensen M (2010 Nov 23). Bovipain-2, the falcipain-2 ortholog, is expressed in intraerythrocytic stages of the tick-transmitted hemoparasite Babesia bovis. Parasites & Vectors. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003645</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3967086">Guo C, Guo R, Xu X, Gao M, Li X, Song J, Zheng Y, Wang X (2014 Feb 7). Evolution and expression analysis of the grape (Vitis vinifera L.) WRKY gene family. Journal of Experimental Botany. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3967086</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4393810">Bruenn JA, Warner BE, Yerramsetty P (2015 Apr 9). Widespread mitovirus sequences in plant genomes. PeerJ. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4393810</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4732000">Qiao Z, Pingault L, Nourbakhsh-Rey M, Libault M (2016 Jan 29). Comprehensive Comparative Genomic and Transcriptomic Analyses of the Legume Genes Controlling the Nodulation Process. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4732000</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3866978">Colasuonno P, Maria MA, Blanco A, Gadaleta A (2013 Dec 5). Description of durum wheat linkage map and comparative sequence analysis of wheat mapped DArT markers with rice and Brachypodium genomes. BMC Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3866978</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3430549">Revanna KV, Munro D, Gao A, Chiu CC, Pathak A, Dong Q (2012 Aug 2). A web-based multi-genome synteny viewer for customized data. BMC Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3430549</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049755">Donoghue MT, Keshavaiah C, Swamidatta SH, Spillane C (2011 Feb 18). Evolutionary origins of Brassicaceae specific genes in Arabidopsis thaliana. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049755</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3618312">Książkiewicz M, Wyrwa K, Szczepaniak A, Rychel S, Majcherkiewicz K, Przysiecka Ł, Karlowski W, Wolko B, Naganowska B (2013 Feb 5). Comparative genomics of Lupinus angustifolius gene-rich regions: BAC library exploration, genetic mapping and cytogenetics. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3618312</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4851370">Gutsche N, Zachgo S (2016 Apr 29). The N-Terminus of the Floral Arabidopsis TGA Transcription Factor PERIANTHIA Mediates Redox-Sensitive DNA-Binding. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4851370</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4109783">Ganapathy G, Howard JT, Ward JM, Li J, Li B, Li Y, Xiong Y, Zhang Y, Zhou S, Schwartz DC, Schatz M, Aboukhalil R, Fedrigo O, Bukovnik L, Wang T, Wray G, Rasolonjatovo I, Winer R, Knight JR, Koren S, Warren WC, Zhang G, Phillippy AM, Jarvis ED (2014 Jul 8). High-coverage sequencing and annotated assemblies of the budgerigar genome. GigaScience. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4109783</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4740955">Pandey A, Misra P, Alok A, Kaur N, Sharma S, Lakhwani D, Asif MH, Tiwari S, Trivedi PK (2016 Feb 1). Genome-Wide Identification and Expression Analysis of Homeodomain Leucine Zipper Subfamily IV (HDZ IV) Gene Family from Musa accuminata. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4740955</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4038794">Rabier CE, Ta T, Ané C (2013 Dec 19). Detecting and Locating Whole Genome Duplications on a Phylogeny: A Probabilistic Approach. Molecular Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4038794</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3970191">Martin JA, Johnson NV, Gross SM, Schnable J, Meng X, Wang M, Coleman-Derr D, Lindquist E, Wei CL, Kaeppler S, Chen F, Wang Z (2014 Mar 31). A near complete snapshot of the Zea mays seedling transcriptome revealed from ultra-deep sequencing. Scientific Reports. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3970191</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531184">Lee TH, Tang H, Wang X, Paterson AH (2012 Nov 23). PGDD: a database of gene and genome duplication in plants. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531184</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4783527">Xu Y, Bi C, Wu G, Wei S, Dai X, Yin T, Ye N (2016 Feb 24). VGSC: A Web-Based Vector Graph Toolkit of Genome Synteny and Collinearity. BioMed Research International. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4783527</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2886086">Kane J, Freeling M, Lyons E (2010 May 22). The Evolution of a High Copy Gene Array in Arabidopsis. Journal of Molecular Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2886086</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2924827">Shi T, Huang H, Barker MS (2010 Jun 24). Ancient genome duplications during the evolution of kiwifruit (Actinidia) and related Ericales. Annals of Botany. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2924827</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3358895">Reneker J, Lyons E, Conant GC, Pires JC, Freeling M, Shyu CR, Korkin D (2012 Apr 10). Long identical multispecies elements in plant and animal genomes. Proceedings of the National Academy of Sciences of the United States of America. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3358895</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5062844">Li W, Chen M, Wang E, Hu L, Hawkesford MJ, Zhong L, Chen Z, Xu Z, Li L, Zhou Y, Guo C, Ma Y (2016 Oct 12). Genome-wide analysis of autophagy-associated genes in foxtail millet (Setaria italica L.) and characterization of the function of SiATG8a in conferring tolerance to nitrogen starvation in rice. BMC Genomics. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5062844</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2814516">Proost S, Van Bel M, Sterck L, Billiau K, Van Parys T, Van de Peer Y, Vandepoele K (2009 Dec). PLAZA: A Comparative Genomics Resource to Study Gene and Genome Evolution in Plants. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2814516</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3326336">Wang Y, Tang H, DeBarry JD, Tan X, Li J, Wang X, Lee TH, Jin H, Marler B, Guo H, Kissinger JC, Paterson AH (2012 Jan 4). MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3326336</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4464145">Wang M, Vannozzi A, Wang G, Zhong Y, Corso M, Cavallini E, Cheng ZM (2015 Jun 12). A comprehensive survey of the grapevine VQ gene family and its transcriptional correlation with WRKY proteins. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4464145</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219971">Zuccolo A, Bowers JE, Estill JC, Xiong Z, Luo M, Sebastian A, Goicoechea JL, Collura K, Yu Y, Jiao Y, Duarte J, Tang H, Ayyampalayam S, Rounsley S, Kudrna D, Paterson AH, Pires JC, Chanderbali A, Soltis DE, Chamala S, Barbazuk B, Soltis PS, Albert VA, Ma H, Mandoli D, Banks J, Carlson JE, Tomkins J, dePamphilis CW, Wing RA, Leebens-Mack J (2011 May 27). A physical map for the Amborella trichopoda genome sheds light on the evolution of angiosperm genome structure. Genome Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3219971</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4783356">Rao X, Lu N, Li G, Nakashima J, Tang Y, Dixon RA (2016 Feb 19). Comparative cell-specific transcriptomics reveals differentiation of C4 photosynthesis pathways in switchgrass and other C4 lineages. Journal of Experimental Botany. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4783356</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383449">Field B, Osbourn A (2012 Jan 1). Order in the playground: Formation of plant gene clusters in dynamic chromosomal regions. Mobile Genetic Elements. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383449</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3316664">Tang H, Woodhouse MR, Cheng F, Schnable JC, Pedersen BS, Conant G, Wang X, Freeling M, Pires JC (2012 Apr). Altered Patterns of Fractionation and Exon Deletions in Brassica rapa Support a Two-Step Model of Paleohexaploidy. Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3316664</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876103">Cheng C, Xu X, Gao M, Li J, Guo C, Song J, Wang X (2013 Dec 12). Genome-Wide Analysis of Respiratory Burst Oxidase Homologs in Grape (Vitis vinifera L.). International Journal of Molecular Sciences. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3876103</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548842">Pavlopoulos GA, Malliarakis D, Papanikolaou N, Theodosiou T, Enright AJ, Iliopoulos I (2015 Aug 25). Visualizing genome and systems biology: technologies, tools, implementation techniques and trends, past, present and future. GigaScience. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548842</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3359998">Albuquerque P, Caridade CM, Rodrigues AS, Marcal AR, Cruz J, Cruz L, Santos CL, Mendes MV, Tavares F (2012 May 24). Evolutionary and Experimental Assessment of Novel Markers for Detection of Xanthomonas euvesicatoria in Plant Samples. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3359998</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5029118">Wang P, Cheng T, Lu M, Liu G, Li M, Shi J, Lu Y, Laux T, Chen J (2016 Sep 20). Expansion and Functional Divergence of AP2 Group Genes in Spermatophytes Determined by Molecular Evolution and Arabidopsis Mutant Analysis. Frontiers in Plant Science. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5029118</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751884">Guo R, Xu X, Carole B, Li X, Gao M, Zheng Y, Wang X (2013 Aug 15). Genome-wide identification, evolutionary and expression analysis of the aspartic protease gene superfamily in grape. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751884</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388744">Liu Y, Wang J, Ge W, Wang Z, Li Y, Yang N, Sun S, Zhang L, Wang X (2017 Apr 12). Two Highly Similar Poplar Paleo-subgenomes Suggest an Autotetraploid Ancestor of Salicaceae Plants. Frontiers in Plant Science. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388744</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263668">Jaratlerdsiri W, Deakin J, Godinez RM, Shan X, Peterson DG, Marthey S, Lyons E, McCarthy FM, Isberg SR, Higgins DP, Chong AY, John JS, Glenn TC, Ray DA, Gongora J (2014 Dec 11). Comparative Genome Analyses Reveal Distinct Structure in the Saltwater Crocodile MHC. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263668</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4875958">Chen M, Tan Q, Sun M, Li D, Fu X, Chen X, Xiao W, Li L, Gao D (2016 Mar 7). Genome-wide identification of WRKY family genes in peach and analysis of WRKY expression during bud dormancy. Molecular Genetics and Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4875958</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3439424">Zhang Y, Gao M, Singer SD, Fei Z, Wang H, Wang X (2012 Sep 11). Genome-Wide Identification and Analysis of the TIFY Gene Family in Grape. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3439424</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5408086">Fan S, Zhang D, Gao C, Zhao M, Wu H, Li Y, Shen Y, Han M (2017 Apr 28). Identification, Classification, and Expression Analysis of GRAS Gene Family in Malus domestica. Frontiers in Physiology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5408086</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4232544">Boyd BM, Allen JM, de Crécy-Lagard V, Reed DL (2014 Sep 11). Genome Sequence of Candidatus Riesia pediculischaeffi, Endosymbiont of Chimpanzee Lice, and Genomic Comparison of Recently Acquired Endosymbionts from Human and Chimpanzee Lice. G3: Genes|Genomes|Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4232544</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073911">Gustin JL, Zanis MJ, Salt DE (2011 Mar 24). Structure and evolution of the plant cation diffusion facilitator family of ion transporters. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073911</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2817432">Barker MS, Vogel H, Schranz ME (2009 Oct 5). Paleopolyploidy in the Brassicales: Analyses of the Cleome Transcriptome Elucidate the History of Genome Duplications in Arabidopsis and Other Brassicales. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2817432</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3902989">Campoli C, Pankin A, Drosse B, Casao CM, Davis SJ, von Korff M (2013 Jun 3). HvLUX1 is a candidate gene underlying the early maturity 10 locus in barley: phylogeny, diversity, and interactions with the circadian clock and photoperiodic pathways. The New Phytologist. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3902989</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355796">Schnable JC, Pedersen BS, Subramaniam S, Freeling M (2011 Mar 10). Dose–Sensitivity, Conserved Non-Coding Sequences, and Duplicate Gene Retention Through Multiple Tetraploidies in the Grasses. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355796</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2893956">Woodhouse MR, Schnable JC, Pedersen BS, Lyons E, Lisch D, Subramaniam S, Freeling M (2010 Jun 29). Following Tetraploidy in Maize, a Short Deletion Mechanism Removed Genes Preferentially from One of the Two Homeologs. PLoS Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2893956</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3641636">Subramaniam S, Wang X, Freeling M, Pires JC (2013 Mar 14). The Fate of Arabidopsis thaliana Homeologous CNSs and Their Motifs in the Paleohexaploid Brassica rapa. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3641636</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4566204">Mohammadin S, Edger PP, Pires JC, Schranz ME (2015 Sep 11). Positionally-conserved but sequence-diverged: identification of long non-coding RNAs in the Brassicaceae and Cleomaceae. BMC Plant Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4566204</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4795695">Mauro-Herrera M, Doust AN (2016 Mar 17). Development and Genetic Control of Plant Architecture and Biomass in the Panicoid Grass, Setaria. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4795695</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4969343">Wyrwa K, Książkiewicz M, Szczepaniak A, Susek K, Podkowiński J, Naganowska B (2016 May 11). Integration of Lupinus angustifolius L. (narrow-leafed lupin) genome maps and comparative mapping within legumes. Chromosome Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4969343</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4013592">Wang H, Yin X, Li X, Wang L, Zheng Y, Xu X, Zhang Y, Wang X (2014 Apr 3). Genome-Wide Identification, Evolution and Expression Analysis of the Grape (Vitis vinifera L.) Zinc Finger-Homeodomain Gene Family. International Journal of Molecular Sciences. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4013592</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4869279">Fister AS, Mejia LC, Zhang Y, Herre EA, Maximova SN, Guiltinan MJ (2016 May 17). Theobroma cacao L. pathogenesis-related gene tandem array members show diverse expression dynamics in response to pathogen colonization. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4869279</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5322546">Hofberger JA, Nsibo DL, Govers F, Bouwmeester K, Schranz ME (2015 Jan 28). A Complex Interplay of Tandem- and Whole-Genome Duplication Drives Expansion of the L-Type Lectin Receptor Kinase Gene Family in the Brassicaceae. Genome Biology and Evolution. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5322546</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3088904">Tang H, Lyons E, Pedersen B, Schnable JC, Paterson AH, Freeling M (2011 Apr 18). Screening synteny blocks in pairwise genome comparisons through integer programming. BMC Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3088904</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3961402">Drillon G, Carbone A, Fischer G (2014 Mar 20). SynChro: A Fast and Easy Tool to Reconstruct and Visualize Synteny Blocks along Eukaryotic Chromosomes. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3961402</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4122759">de Setta N, Monteiro-Vitorello CB, Metcalfe CJ, Cruz GM, Del Bem LE, Vicentini R, Nogueira FT, Campos RA, Nunes SL, Turrini PC, Vieira AP, Ochoa Cruz EA, Corrêa TC, Hotta CT, de Mello Varani A, Vautrin S, da Trindade AS, de Mendonça Vilela M, Lembke CG, Sato PM, de Andrade RF, Nishiyama MY Jr, Cardoso-Silva CB, Scortecci KC, Garcia AA, Carneiro MS, Kim C, Paterson AH, Bergès H, D’Hont A, de Souza AP, Souza GM, Vincentz M, Kitajima JP, Van Sluys MA (2014 Jun 30). Building the sugarcane genome for biotechnology and identifying evolutionary trends. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4122759</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280228">Zhang Y, Mao L, Wang H, Brocker C, Yin X, Vasiliou V, Fei Z, Wang X (2012 Feb 15). Genome-Wide Identification and Analysis of Grape Aldehyde Dehydrogenase (ALDH) Gene Superfamily. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280228</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4119204">Cannarozzi G, Plaza-Wüthrich S, Esfeld K, Larti S, Wilson YS, Girma D, de Castro E, Chanyalew S, Blösch R, Farinelli L, Lyons E, Schneider M, Falquet L, Kuhlemeier C, Assefa K, Tadele Z (2014 Jul 9). Genome and transcriptome sequencing identifies breeding targets in the orphan crop tef (Eragrostis tef). BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4119204</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059761">Liu Z, Tavares R, Forsythe ES, André F, Lugan R, Jonasson G, Boutet-Mercey S, Tohge T, Beilstein MA, Werck-Reichhart D, Renault H (2016 Oct 7). Evolutionary interplay between sister cytochrome P450 genes shapes plasticity in plant metabolism. Nature Communications. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059761</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4199706">Trujillo DI, Silverstein KA, Young ND (2014 Aug 25). Genomic Characterization of the LEED..PEEDs, a Gene Family Unique to the Medicago Lineage. G3: Genes|Genomes|Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4199706</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3095355">Sanzol J (2010 May 14). Dating and functional characterization of duplicated genes in the apple (Malus domestica Borkh.) by analyzing EST data. BMC Plant Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3095355</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4239561">Chen EC, Sankoff D (2014 Oct 17). Gene expression and fractionation resistance. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4239561</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258165">Abeel T, Van Parys T, Saeys Y, Galagan J, Van de Peer Y (2011 Nov 17). GenomeView: a next-generation genome browser. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258165</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4700967">Tang H, Bomhoff MD, Briones E, Zhang L, Schnable JC, Lyons E (2015 Nov 11). SynFind: Compiling Syntenic Regions across Any Set of Genomes on Demand. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4700967</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053395">Schnable JC, Freeling M (2011 Mar 10). Genes Identified by Visible Mutant Phenotypes Show Increased Bias toward One of Two Subgenomes of Maize. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3053395</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4811886">Zhao J, Guo R, Guo C, Hou H, Wang X, Gao H (2016 Mar 30). Evolutionary and Expression Analyses of the Apple Basic Leucine Zipper Transcription Factor Family. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4811886</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3845156">Waters AJ, Bilinski P, Eichten SR, Vaughn MW, Ross-Ibarra J, Gehring M, Springer NM (2013 Nov 11). Comprehensive analysis of imprinted genes in maize reveals allelic variation for imprinting and limited conservation with other species. Proceedings of the National Academy of Sciences of the United States of America. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3845156</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045373">Lyons E, Freeling M, Kustu S, Inwood W (2011 Feb 25). Using Genomic Sequencing for Classical Genetics in E. coli K12. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045373</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5146710">Mykles DL, Burnett KG, Durica DS, Joyce BL, McCarthy FM, Schmidt CJ, Stillman JH (2016 Sep 17). Resources and Recommendations for Using Transcriptomics to Address Grand Challenges in Comparative Biology. Integrative and Comparative Biology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5146710</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045888">Mondego JM, Vidal RO, Carazzolle MF, Tokuda EK, Parizzi LP, Costa GG, Pereira LF, Andrade AC, Colombo CA, Vieira LG, Pereira GA (2011 Feb 8). An EST-based analysis identifies new genes and reveals distinctive gene expression features of Coffea arabica and Coffea canephora. BMC Plant Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045888</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3430884">Cheng F, Wu J, Fang L, Wang X (2012 Aug 30). Syntenic gene analysis between Brassica rapa and other Brassicaceae species. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3430884</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408644">Tang H, Lyons E (2012 Jul 31). Unleashing the Genome of Brassica Rapa. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408644</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679924">Morioka K, Yockteng R, Almeida AM, Specht CD (2015 Dec 16). Loss of YABBY2-Like Gene Expression May Underlie the Evolution of the Laminar Style in Canna and Contribute to Floral Morphological Diversity in the Zingiberales. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679924</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105427">Soderlund C, Bomhoff M, Nelson WM (2011 Mar 11). SyMAP v3.4: a turnkey synteny system with application to plant genomes. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105427</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3708275">Turco G, Schnable JC, Pedersen B, Freeling M (2013 Jul 2). Automated conserved non-coding sequence (CNS) discovery reveals differences in gene content and promoter evolution among grasses. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3708275</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289383">Hofberger JA, Zhou B, Tang H, Jones JD, Schranz ME (2014 Nov 8). A novel approach for multi-domain and multi-gene family identification provides insights into evolutionary dynamics of disease resistance genes in core eudicot plants. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289383</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4240082">Zheng C, Kononenko A, Leebens-Mack J, Lyons E, Sankoff D (2014 Oct 17). Gene families as soft cliques with backbones: Amborella contrasted with other flowering plants. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4240082</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3436819">Sankoff D, Zheng C (2012 Sep 3). Fractionation, rearrangement and subgenome dominance. Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3436819</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4397884">Berke L, Snel B (2015 Mar 13). The plant Polycomb repressive complex 1 (PRC1) existed in the ancestor of seed plants and has a complex duplication history. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4397884</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3537189">Yang J, Osman K, Iqbal M, Stekel DJ, Luo Z, Armstrong SJ, Franklin FC (2013 Jan 4). Inferring the Brassica rapa Interactome Using Protein–Protein Interaction Data from Arabidopsis thaliana. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3537189</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3095360">Hand ML, Cogan NO, Sawbridge TI, Spangenberg GC, Forster JW (2010 May 24). Comparison of homoeolocus organisation in paired BAC clones from white clover (Trifolium repens L.) and microcolinearity with model legume species. BMC Plant Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3095360</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4457800">Hofberger JA, Ramirez AM, van den Bergh E, Zhu X, Bouwmeester HJ, Schuurink RC, Schranz ME (2015 Jun 5). Large-Scale Evolutionary Analysis of Genes and Supergene Clusters from Terpenoid Modular Pathways Provides Insights into Metabolic Diversification in Flowering Plants. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4457800</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4603330">Zheng C, Santos Muñoz D, Albert VA, Sankoff D (2015 Oct 2). Syntenic block overlap multiplicities with a panel of reference genomes provide a signature of ancient polyploidization events. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4603330</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3814328">Froyd CA, Kapoor S, Dietrich F, Rusche LN (2013 Oct 31). The Deacetylase Sir2 from the Yeast Clavispora lusitaniae Lacks the Evolutionarily Conserved Capacity to Generate Subtelomeric Heterochromatin. PLoS Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3814328</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3564988">Mauro-Herrera M, Wang X, Barbier H, Brutnell TP, Devos KM, Doust AN (2013 Feb 1). Genetic Control and Comparative Genomic Analysis of Flowering Time in Setaria (Poaceae). G3: Genes|Genomes|Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3564988</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3240960">Hudson CM, Puckett EE, Bekaert M, Pires JC, Conant GC (2011 Nov 4). Selection for Higher Gene Copy Number after Different Types of Plant Gene Duplications. Genome Biology and Evolution. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3240960</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271752">Van Bel M, Proost S, Wischnitzki E, Movahedi S, Scheerlinck C, Van de Peer Y, Vandepoele K (2011 Dec 23). Dissecting Plant Genomes with the PLAZA Comparative Genomics Platform. Plant Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271752</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4295026">Książkiewicz M, Zielezinski A, Wyrwa K, Szczepaniak A, Rychel S, Karlowski W, Wolko B, Naganowska B (2014 May 15). Remnants of the Legume Ancestral Genome Preserved in Gene-Rich Regions: Insights from Lupinus angustifolius Physical, Genetic, and Comparative Mapping. Plant Molecular Biology Reporter / Ispmb. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4295026</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3481203">Wiszniewski AA, Smith SM, Bussell JD (2012 Oct 15). Conservation of two lineages of peroxisomal (Type I) 3-ketoacyl-CoA thiolases in land plants, specialization of the genes in Brassicaceae, and characterization of their expression in Arabidopsis thaliana. Journal of Experimental Botany. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3481203</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355756">Goff SA, Vaughn M, McKay S, Lyons E, Stapleton AE, Gessler D, Matasci N, Wang L, Hanlon M, Lenards A, Muir A, Merchant N, Lowry S, Mock S, Helmke M, Kubach A, Narro M, Hopkins N, Micklos D, Hilgert U, Gonzales M, Jordan C, Skidmore E, Dooley R, Cazes J, McLay R, Lu Z, Pasternak S, Koesterke L, Piel WH, Grene R, Noutsos C, Gendler K, Feng X, Tang C, Lent M, Kim SJ, Kvilekval K, Manjunath BS, Tannen V, Stamatakis A, Sanderson M, Welch SM, Cranston KA, Soltis P, Soltis D, O\'Meara B, Ane C, Brutnell T, Kleibenstein DJ, White JW, Leebens-Mack J, Donoghue MJ, Spalding EP, Vision TJ, Myers CR, Lowenthal D, Enquist BJ, Boyle B, Akoglu A, Andrews G, Ram S, Ware D, Stein L, Stanzione D (2011 Jul 25). The iPlant Collaborative: Cyberinfrastructure for Plant Biology. Frontiers in plant science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355756</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5073747">Książkiewicz M, Rychel S, Nelson M. N., Wyrwa K, Naganowska B, Wolko B (2016 Oct 21). Expansion of the phosphatidylethanolamine binding protein family in legumes: a case study of Lupinus angustifolius L. FLOWERING LOCUS T homologs, LanFTc1 and LanFTc2. BMC Genomics. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5073747</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679983">Devis D, Firth SM, Liang Z, Byrne ME (2015 Dec 16). Dosage Sensitivity of RPL9 and Concerted Evolution of Ribosomal Protein Genes in Plants. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679983</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3951882">Jung S, Main D (2013 May 21). Genomics and bioinformatics resources for translational science in Rosaceae. Plant Biotechnology Reports. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3951882</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4404975">Przysiecka Ł, Książkiewicz M, Wolko B, Naganowska B (2015 Apr 21). Structure, expression profile and phylogenetic inference of chalcone isomerase-like genes from the narrow-leafed lupin (Lupinus angustifolius L.) genome. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4404975</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152334">Reineke AR, Bornberg-Bauer E, Gu J (2011 Apr 5). Evolutionary divergence and limits of conserved non-coding sequence detection in plant genomes. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152334</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4030238">Reichardt N, Duncan SH, Young P, Belenguer A, McWilliam Leitch C, Scott KP, Flint HJ, Louis P (2014 Feb 20). Phylogenetic distribution of three pathways for propionate production within the human gut microbiota. The ISME Journal. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4030238</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4895484">Zheng C, Sankoff D (2016 Jan 11). Locating rearrangement events in a phylogeny based on highly fragmented assemblies. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4895484</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3474763">Chen L, Hu W, Tan S, Wang M, Ma Z, Zhou S, Deng X, Zhang Y, Huang C, Yang G, He G (2012 Oct 17). Genome-Wide Identification and Analysis of MAPK and MAPKK Gene Families in Brachypodium distachyon. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3474763</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4595791">Haug-Baltzell A, Jarvis ED, McCarthy FM, Lyons E (2015 Oct 7). Identification of dopamine receptors across the extant avian family tree and analysis with other clades uncovers a polyploid expansion among vertebrates. Frontiers in Neuroscience. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4595791</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4726258">Scaglione D, Reyes-Chin-Wo S, Acquadro A, Froenicke L, Portis E, Beitel C, Tirone M, Mauro R, Lo Monaco A, Mauromicale G, Faccioli P, Cattivelli L, Rieseberg L, Michelmore R, Lanteri S (2016 Jan 20). The genome sequence of the outbreeding globe artichoke constructed de novo incorporating a phase-aware low-pass sequencing strategy of F1 progeny. Scientific Reports. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4726258</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4034038">Coman D, Altenhoff A, Zoller S, Gruissem W, Vranová E (2014 May 27). Distinct evolutionary strategies in the GGPPS family from plants. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4034038</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959230">Boyd BM, Allen JM, Koga R, Fukatsu T, Sweet AD, Johnson KP, Reed DL (2016 May 16). Two Bacterial Genera, Sodalis and Rickettsia, Associated with the Seal Louse Proechinophthirus fluctus (Phthiraptera: Anoplura). Applied and Environmental Microbiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959230</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531202">Nussbaumer T, Martis MM, Roessner SK, Pfeifer M, Bader KC, Sharma S, Gundlach H, Spannagl M (2012 Nov 29). MIPS PlantsDB: a database framework for comparative plant genome research. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531202</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851865">Zheng C, Sankoff D (2013 Oct 15). Practical aliquoting of flowering plant genomes. BMC Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851865</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042036">Kang YJ, Lee T, Lee J, Shim S, Jeong H, Satyawan D, Kim MY, Lee S (2015 Aug 13). Translational genomics for plant breeding with the genome sequence explosion. Plant Biotechnology Journal. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042036</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5370463">Rendón-Anaya M, Montero-Vargas JM, Saburido-Álvarez S, Vlasova A, Capella-Gutierrez S, Ordaz-Ortiz JJ, Aguilar OM, Vianello-Brondani RP, Santalla M, Delaye L, Gabaldón T, Gepts P, Winkler R, Guigó R, Delgado-Salinas A, Herrera-Estrella A (2017 Mar 29). Genomic history of the origin and domestication of common bean unveils its closest sister species. Genome Biology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5370463</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4315028">Zhang J, Wang X, Cheng F, Wu J, Liang J, Yang W, Wang X (2015 Feb 3). Lineage-specific evolution of Methylthioalkylmalate synthases (MAMs) involved in glucosinolates biosynthesis. Frontiers in Plant Science. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4315028</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389459">Zheng C, Sankoff D (2012 Jun 25). Gene order in rosid phylogeny, inferred from pairwise syntenies among extant genomes. BMC Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3389459</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3968010">Hyung D, Lee C, Kim JH, Yoo D, Seo YS, Jeong SC, Lee JH, Chung Y, Jung KH, Cook DR, Choi HK (2014 Mar 27). Cross-Family Translational Genomics of Abiotic Stress-Responsive Genes between Arabidopsis and Medicago truncatula. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3968010</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3817805">Zheng C, Chen E, Albert VA, Lyons E, Sankoff D (2013 Nov 5). Ancient eudicot hexaploidy meets ancestral eurosid gene order. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3817805</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3576282">Qin Z, Cai Z, Xia G, Wang M (2013 Jan 28). Synonymous codon usage bias is correlative to intron number and shows disequilibrium among exons in plants. BMC Genomics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3576282</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4308911">Masonbrink RE, Gallagher JP, Jareczek JJ, Renny-Byfield S, Grover CE, Gong L, Wendel JF (2014 Dec 30). CenH3 evolution in diploids and polyploids of three angiosperm genera. BMC Plant Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4308911</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4646352">Pereira-Santana A, Alcaraz LD, Castaño E, Sanchez-Calderon L, Sanchez-Teyer F, Rodriguez-Zapata L (2015 Nov 16). Comparative Genomics of NAC Transcriptional Factors in Angiosperms: Implications for the Adaptation and Diversification of Flowering Plants. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4646352</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3958358">Guimaraes AM, Santos AP, do Nascimento NC, Timenetsky J, Messick JB (2014 Mar 18). Comparative Genomics and Phylogenomics of Hemotrophic Mycoplasmas. PLoS ONE. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3958358</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4102449">Delaux PM, Varala K, Edger PP, Coruzzi GM, Pires JC, Ané JM (2014 Jul 17). Comparative Phylogenomics Uncovers the Impact of Symbiotic Associations on Host Genome Evolution. PLoS Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4102449</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3491363">Ranjan A, Ichihashi Y, Sinha NR (2012 Aug 30). The tomato genome: implications for plant breeding, genomics and evolution. Genome Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3491363</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003123">Baek JH, Kim J, Kim CK, Sohn SH, Choi D, Ratnaparkhe MB, Kim DW, Lee TH (2016 Aug 28). MultiSyn: A Webtool for Multiple Synteny Detection and Visualization of User’s Sequence of Interest Compared to Public Plant Species. Evolutionary Bioinformatics Online. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003123</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3850447">Mühlhausen S, Kollmar M (2013 Sep 22). Whole genome duplication events in plant evolution reconstructed and predicted using myosin motor proteins. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3850447</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987669">Wu W, Yang YL, He WM, Rouard M, Li WM, Xu M, Roux N, Ge XJ (2016 Aug 17). Whole genome sequencing of a banana wild relative Musa itinerans provides insights into lineage-specific diversification of the Musa genus. Scientific Reports. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987669</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5015945">Nelson AD, Forsythe ES, Devisetty UK, Clausen DS, Haug-Batzell AK, Meldrum AM, Frank MR, Lyons E, Beilstein MA (2016 Jul 15). A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants. G3: Genes|Genomes|Genetics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5015945</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068949">Rupert CB, Heltzel JM, Taylor DJ, Rusche LN (2016 Aug 18). Sporadic Gene Loss After Duplication Is Associated with Functional Divergence of Sirtuin Deacetylases Among Candida Yeast Species. G3: Genes|Genomes|Genetics. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068949</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852042">Chen EC, Buen Abad Najar CF, Zheng C, Brandts A, Lyons E, Tang H, Carretero-Paulet L, Albert VA, Sankoff D (2013 Oct 15). The dynamics of functional classes of plant genes in rediploidized ancient polyploids. BMC Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852042</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4301744">Louis A, Murat F, Salse J, Roest Crollius H (2014 Nov 27). GenomicusPlants: A Web Resource to Study Genome Evolution in Flowering Plants. Plant and Cell Physiology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4301744</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853686">Muffato M, Louis A, Poisnel CE, Crollius HR (2010 Feb 24). Genomicus: a database and a browser to study gene synteny in modern and ancestral genomes. Bioinformatics. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853686</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4384038">Proost S, Van\xa0Bel M, Vaneechoutte D, Van\xa0de\xa0Peer Y, Inzé D, Mueller-Roeber B, Vandepoele K (2014 Oct 16). PLAZA 3.0: an access point for plant comparative genomics. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4384038</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987893">Lee J, Hong WY, Cho M, Sim M, Lee D, Ko Y, Kim J (2016 May 6). Synteny Portal: a web-based application portal for synteny block analysis. Nucleic Acids Research. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987893</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4145113">Kagale S, Robinson SJ, Nixon J, Xiao R, Huebert T, Condie J, Kessler D, Clarke WE, Edger PP, Links MG, Sharpe AG, Parkin IA (2014 Jul 17). Polyploid Evolution of the Brassicaceae during the Cenozoic Era. The Plant Cell. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4145113</a>', '<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4928308">Yerramsetty P, Stata M, Siford R, Sage TL, Sage RF, Wong GK, Albert VA, Berry JO (2016 Jun 29). Evolution of RLSB, a nuclear-encoded S1 domain RNA binding protein associated with post-transcriptional regulation of plastid-encoded rbcL mRNA in vascular plants. BMC Evolutionary Biology. Retrieved from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4928308</a>', '<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5084476">Studer AJ, Schnable JC, Weissmann S, Kolbe AR, McKain MR, Shao Y, Cousins AB, Kellogg EA, Brutnell TP (2016 Oct 28). The draft genome of the C3 panicoid grass species Dichanthelium oligosanthes. Genome Biology. Retrieved from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5084476</a>'],
        marker: {
            color: ['rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(244, 241, 48)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)', 'rgb(8, 114, 32)']
        },
        type: 'bar'
            }];

                layout = {
                  title: 'Cosine Similarity to On The Origin Of Species (%)',
                    margin: {
                        b: 160
                    }
                };


            Plotly.newPlot('myDiv', data, layout);

        myPlot.on('plotly_click', function(data){
        for (var i=0; i<data.points.length; i++) { //safety net
            var selected_point = data.points[i];
    //        console.log(selected_point)

            var point_number = data.points[i].pointNumber;
            var point_data = selected_point['data'];
            var point_title = point_data['name'][point_number];
    //		var point_title = data.points[i]['data']['name'][data.points[i].pointNumber];
    //		console.log(point_title);

            clickInfo.innerHTML = point_title;
        }
    });  //end plotly_click

</script>



 <!-- Progress modal -->
<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
        <h4 class="modal-title" id="myModalLabel">Loading</h4>
      </div>
      <div class="modal-body">

				<!-- Progress bar that doesn't actually move -->
				<div class="progress">
				  <div class="progress-bar progress-bar-success progress-bar-striped active" role="progressbar" aria-valuenow="1" aria-valuemin="0" aria-valuemax="100" style="width: 45%">
				    <span class="sr-only">45% Complete</span>
				  </div>
				</div>

      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
      </div>
    </div>
  </div>
</div>


</body>




<script>
  var btn = document.getElementById('litbutton');
  btn.addEventListener('click', loadProgress);

  function loadProgress() {
    //console.log('Button clicked!');
    var progressBar = document.getElementsByClassName('progress-bar')[0];
    var interval = 30;
    var currentValue = 0;
    for (var i = 0; i < 100; i += interval) {
      //console.log('Setting interval', i)
      setTimeout(function() {
        currentValue += i;
        //console.log('Setting progress width to:', currentValue, '%');
        progressBar.setAttribute('style', 'width: ' + currentValue + '%');
      }, i);
    }
  }
</script>